src/main/java/echoquery/utils/VisualizationUtil.java

```diff
@@ -38,4 +43,26 @@ public static String getDisplayText() {
       return "There was an error retrieving the display text from the database";
     }
   }
-}
+  
+  // Since id is a unique column in the sessions table, we don't need to worry
+  // about duplicate rows being inserted
+  public static void makeSureSessionExistsInDB(String userId) {
+      try {
+        Statement statement = conn.createStatement();
+        statement.executeUpdate("insert into sessions values ('" + 
+            cleanId(userId) + "', 'EchoQuery. What do you want?');");
+      } catch (SQLException e) {
+        // silently ignore - we already have this value in the db
+      }
+  }
+  
+  // removes the text at the beginning of the id
+  public static String cleanId(String userId) {
```

Comment:
the ids are by default formatted like amazon.user.KJEFLKDSJBDFSK or something like that so I snip off the first two default words